rest/route/service.go

```diff
@@ -54,7 +54,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/admin/task_queue").Version(2).Delete().Wrap(superUser).RouteHandler(makeClearTaskQueueHandler(sc))
 	app.AddRoute("/admin/commit_queues").Version(2).Delete().Wrap(superUser).RouteHandler(makeClearCommitQueuesHandler(sc))
 	app.AddRoute("/alias/{name}").Version(2).Get().RouteHandler(makeFetchAliases(sc))
-	app.AddRoute("/builds/{build_id}").Version(2).Get().RouteHandler(makeGetBuildByID(sc))
+	app.AddRoute("/builds/{build_id}").Version(2).Get().Wrap(RequiresProjectPermission(evergreen.PermissionTasks, evergreen.TasksView)).RouteHandler(makeGetBuildByID(sc))
```

Comment:
can we have the middlewares be defined somewhere, centrally (like checkUser/superUser) rather than constructed a bunch of times?